src/PhpSpec/Console/IO.php

```diff
@@ -13,15 +13,15 @@
 
 namespace PhpSpec\Console;
 
-use PhpSpec\IO\IOInterface;
+use PhpSpec\IO\IOStream;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use PhpSpec\Config\OptionsConfig;
 
 /**
  * Class IO deals with input and output from command line interaction
  */
-class IO implements IOInterface
+class IO implements IOStream
```

Comment:
The interface should probably be `IO`, which I guess leaves the question of what is noteworthy about the implementation.